src/features/device-logs/lib/store.ts

```diff
@@ -52,27 +52,27 @@ const getWriteContext = (() => {
 			uuid: string,
 			req: permissions.PermissionReq,
 		): Promise<false | LogContext> => {
-			return await sbvrUtils.db.readTransaction(async (tx) => {
-				try {
-					const result = await api.resin.request({
+			try {
+				const result = await sbvrUtils.db.readTransaction(async (tx) => {
```

Comment:
I guess we don't even need to open the TX manually (just checked that a POST to canAccess creates a readTx anyway)